src/pages/index.tsx

```diff
@@ -487,14 +487,6 @@ const Home = observer(() => {
                         </div>
                     ) : (
                         <div className="card" style={{ boxShadow: 'var(--ifm-global-shadow-md)' }}>
```

Comment:
```suggestion
                        <div className="card" style={{ boxShadow: 'var(--ifm-global-shadow-md)' }}>
                          <div className="card__header">
                                <h2>Phase d'introduction</h2>
                            </div>
                            <div className="card__body">
                                Le calendrier est encore dans sa phase d'introduction. Des retours peuvent être transmis
                                grâce à l'application de traçage de bugs (voir Video).
                                <h3>Merci beaucoup pour votre aide.</h3>
                            </div>
```

Reply:
i'd keep this section to make sure users are aware of tha fact that the calendar is still in a introductory phase...